contracts/extensions/StreamingFeeSplitExtension.sol

```diff
@@ -0,0 +1,226 @@
+/*
+    Copyright 2022 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+pragma experimental "ABIEncoderV2";
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import { PreciseUnitMath } from "@setprotocol/set-protocol-v2/contracts/lib/PreciseUnitMath.sol";
+import { ISetToken } from "@setprotocol/set-protocol-v2/contracts/interfaces/ISetToken.sol";
+
+import { BaseGlobalExtension } from "../lib/BaseGlobalExtension.sol";
+import { IManagerCore } from "../interfaces/IManagerCore.sol";
+import { IDelegatedManager } from "../interfaces/IDelegatedManager.sol";
+
+import { IStreamingFeeModule } from "../interfaces/IStreamingFeeModuleV2.sol";
+
+/**
+ * @title StreamingFeeSplitExtension
+ * @author Set Protocol
+ *
+ * Smart contract global extension which provides DelegatedManager owner and 
+ * methodologist the ability to accrue and split streaming fees at an mutable percentage.
+ */
+contract StreamingFeeSplitExtension is BaseGlobalExtension {
+    using Address for address;
+    using PreciseUnitMath for uint256;
+    using SafeMath for uint256;
+
+    /* ============ Events ============ */
+
+    event ExtensionInitialized(
```

Comment:
More specific event names here maybe:
```
StreamingFeeSplitExtensionInitialized
```

And same question `indexed` for the address values...maybe we should discuss as a team whether or not to follow this convention. 

I could be useful to filter events for an activity feed or something, but maybe this evm feature is irrelevant if everything is graphql'd